recipes/polyhedral-gravity-model/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "polyhedral-gravity-model" %}
+{% set org = "esa" %}
+{% set version = "1.1.0" %}
+{% set sha256 = "2255b2cd136cfd0e7f02bb1020f67af4ca2c99e23e4006e18077e10dc0a326ac" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ org }}/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  script_env:
+    - CMAKE_GENERATOR=Ninja
+    - PARALLELIZATION_DEVICE=OMP
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [linux]
+    - ninja
+    - python
+    - spdlog
+    - yaml-cpp
+    - xsimd
+    - pybind11
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/esa/polyhedral-gravity-model
+  summary: 'Simple and fast evaluation of the polyhedral gravity model for a constant density polyhedron'
+  description: |
+    The package polyhedral_gravity provides a simple to use interface for the evaluation of the full gravity
+    tensor of a constant density polyhedron at given computation points. It is based on a fast, parallelized
+    backbone in C++ capable of evaluating the gravity at thousands of computation points in the fraction of a second.
+  license: GPL-3.0-only
+  license_family: GPL3
+  license_file: LICENSE
+  doc_url: https://polyhedral-gravity-model-cpp.readthedocs.io/en/latest
+  dev_url: https://github.com/esa/polyhedral-gravity-model
+
+extra:
+  recipe-maintainers:
+    - schuhmaj
+    - gomezzz
+    - darioizzo
```

Comment:
Those users haven't confirmed they want to be maintainers.

Reply:
I confirm! @beenje

Reply:
Same, also confirm 🙏 @beenje